draft/overlay/modmypi-star.md

```diff
@@ -0,0 +1,163 @@
+<!--
+---
+name: ModMyPi Christmas Tree Star
+class: board
+type: LED
+formfactor: Custom
+manufacturer: ModMyPi
+description: A star shaped LED add-on board for the Raspberry Pi which can be used as a Christmas Tree topper.
+url: https://www.modmypi.com/blog/christmas-tree-star-guide
+github: https://github.com/modmypi/Programmable-Christmas-Star
+buy: https://www.modmypi.com/raspberry-pi-christmas-tree-star
+image: 'modmypi-star.png'
+pincount: 40
+eeprom: no
+power:
+  '2':
+ground:
+  '6':
+  '9':
+  '14':
+  '20':
+  '25':
+  '30':
+  '34':
+  '39':
+pin:
+  '3':
+    name: LED Inner
```

Comment:
I did a bit of poking around, and it seems this actually controls the 5 inner LEDs all at once? Perhaps "Inner LEDs" would be a better name than "LED Inner"? :man_shrugging:

Reply:
I think I was trying to match it to the GPIO Zero extension I wrote in the library which uses star.inner to control the LEDs... Happy to change it if it makes more sense reading it as Inner LEDs.

Reply:
I'll leave it up to @Gadgetoid as an editor's decision :wink:

Reply:
I'm going with plan C - remove the word "LED" from in front of each pin name because we know they're all LEDs.